From 9e8131a96362e5a454faa57459866227715d1b62 Mon Sep 17 00:00:00 2001
From: Jiyong Park <jiyong@google.com>
Date: Mon, 9 Aug 2021 09:47:37 +0900
Subject: [PATCH] Remove ndk_platform backend. Use the ndk backend.

The ndk_platform backend will soon be deprecated because the ndk backend
can serve the same purpose. This is to eliminate the confusion about
having two variants (ndk and ndk_platform) for the same ndk backend.

Bug: 161456198
Test: m
Merged-In: I0652f1efe920111d79dd8c2ecd52e55dda54538f
Change-Id: Ib58c9f1cb80d083a3c62d03415610be855b80f03
(cherry picked from commit c426c6dbdc8db86cc35d15cca6fb2b703280e4c2)
---
 aidl/Android.bp | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/aidl/Android.bp b/aidl/Android.bp
index f88a983..ce31f9d 100644
--- a/aidl/Android.bp
+++ b/aidl/Android.bp
@@ -14,7 +14,7 @@ cc_library_shared {
         "liblog",
         "libqtivibratoreffect",
         "libbinder_ndk",
-        "android.hardware.vibrator-V1-ndk_platform",
+        "android.hardware.vibrator-V1-ndk",
     ],
     export_include_dirs: ["include"]
 }
@@ -36,7 +36,7 @@ cc_binary {
         "libutils",
         "libbase",
         "libbinder_ndk",
-        "android.hardware.vibrator-V1-ndk_platform",
+        "android.hardware.vibrator-V1-ndk",
         "vendor.qti.hardware.vibrator.impl",
     ],
 }
@@ -56,7 +56,7 @@ cc_library_shared {
         "libqtivibratoreffect",
         "libqtivibratoreffectoffload",
         "libbinder_ndk",
-        "android.hardware.vibrator-V1-ndk_platform",
+        "android.hardware.vibrator-V1-ndk",
     ],
     export_include_dirs: ["include"],
 }
@@ -78,7 +78,7 @@ cc_binary {
         "libutils",
         "libbase",
         "libbinder_ndk",
-        "android.hardware.vibrator-V1-ndk_platform",
+        "android.hardware.vibrator-V1-ndk",
         "vendor.qti.hardware.vibrator.offload.impl",
     ],
 }
-- 
2.30.2

